CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
 Contributing to Jbuilder
 =====================
 
-[![Build Status](https://api.travis-ci.org/rails/jbuilder.svg?branch=master)][travis]
+[![Build Status](https://github.com/rails/jbuilder/workflows/Ruby%20test/badge.svg)][test]
 [![Gem Version](https://badge.fury.io/rb/jbuilder.svg)][gem]
 [![Code Climate](https://codeclimate.com/github/rails/jbuilder/badges/gpa.svg)][codeclimate]
 
-[travis]: https://travis-ci.org/rails/jbuilder
+[test]: https://github.com/rails/webpacker/actions?query=branch%3Amaster
```

Comment:
it shows `webpacker` this line. possibly `jbuilder` I'm guessing

Reply:
Fixed in b1894ef.